packages/json-serializer/test/serializeJSON-test.js

```diff
@@ -22,11 +22,18 @@ describe('#serializeJSON', () => {
     const element = new namespace.elements.Element();
     element.element = 'message';
 
+    const dataStructure = new namespace.elements.DataStructure();
```

Comment:
I'm not sure this tests the intended change because its still passing in `element` to `serializeJSON` instead of `dataStructure`. Perhaps it would be better to create a more specific test for it too like in https://github.com/apiaryio/api-elements.js/pull/544/commits/05c0c1e58d4d97ccc68d987f742a5142f8039e30

Reply:
You are right. I added that scenario [here](https://github.com/apiaryio/api-elements.js/pull/540/commits/e570f3f8d4af57c6fcc090fc75e87e5b6fed2aab#diff-58e8df6f0d1b33d31ce6d546934715f8R41).